static/app/components/modals/commandPalette.spec.jsx

```diff
@@ -68,56 +74,42 @@ describe('Command Palette Modal', function () {
   });
 
   it('can open command palette modal and search', async function () {
-    const wrapper = mountWithTheme(
-      <App params={{orgId: 'org-slug'}}>{<div>placeholder content</div>}</App>,
-      TestStubs.routerContext([
+    render(<App params={{orgId: 'org-slug'}}>{<div>placeholder content</div>}</App>, {
+      context: TestStubs.routerContext([
         {
           router: TestStubs.router({
             params: {orgId: 'org-slug'},
           }),
         },
-      ])
-    );
-
-    // No Modal
-    expect(wrapper.find('Modal')).toHaveLength(0);
+      ]),
+    });
 
+    expect(screen.queryByRole('dialog')).not.toBeInTheDocument();
     openCommandPalette({params: {orgId: 'org-slug'}});
```

Comment:
this parameters we are passing is wrong . Do you think you can remove it? I caught this by converting this file to typescript